ng2-components/ng2-activiti-processlist/src/components/activiti-filters.component.html

```diff
@@ -3,7 +3,7 @@
         <li class="mdl-list__item activiti-filters__entry" (click)="selectFilter(filter)" *ngFor="let filter of filters"
             [class.active]="currentFilter === filter">
             <span class="mdl-list__item-primary-content">
-            <i class="material-icons mdl-list__item-icon activiti-filters__entry-icon" [attr.data-automation-id]="filter.name + '_filter'" >assignment</i>
+            <i *ngIf="hasIcon" class="material-icons mdl-list__item-icon activiti-filters__entry-icon" [attr.data-automation-id]="filter.name + '_filter'" >assignment</i>
```

Comment:
Hi @eromano ,

"hasIcon" we got it from ng2-activiti-tasklist/src/components/activiti-filters.component.html. AD F is using this verb in tasklist filter component thatswhy we also followed same. Can you please confirm still we need to change this or not.